src/OrchardCore/OrchardCore.Users.Abstractions/Services/IUserToExternalLoginProvider.cs

```diff
@@ -0,0 +1,49 @@
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Identity;
+
+namespace OrchardCore.Users;
+
+/// <summary>
+/// Interface <c>IUserToExternalLoginProvider</c> allow to create services used to decide
+/// when link an existing local user account to the login informations that comes from an external
+/// login system like OpenId or GitHub.
+/// </summary>
+public interface IUserToExternalLoginProvider
+{
+    /// <summary>
+    /// Method <c>CanHandle</c> extabilish if the service that implement this interface can handle
+    /// a kind of external login.
+    /// </summary>
+    /// <param name="info">
+    /// external login information with type and other data to extabilish if
+    /// service can handle this kind of external login.
+    /// </param>
+    /// <returns>
+    /// True, if the service can handle the external login by the informations in the parameter. False, instead.
+    /// </returns>
```

Comment:
```suggestion
    /// <summary>
    /// Checks if the implementation can handle the provided external login information.
    /// </summary>
    /// <param name="info">The external login information.</param>
    /// <returns>
    /// Returns true if the service can handle the provided external login information; otherwise, returns false.
    /// </returns>
```